tomcat/content/thredds/palettes/revrainbow.pal

```diff
@@ -0,0 +1,27 @@
+102 0 0  #dark red
```

Comment:
Got this be reversing colors here:
https://github.com/Unidata/tds/blob/master/tds/src/main/webapp/WEB-INF/palettes/alg2.pal